src/Ng/Jobs/BackupV3Storage.cs

```diff
@@ -0,0 +1,236 @@
+            await TryCopyContainerAsync(_registrationStorageContainer, currentDate, _registrationStorageAccountName, _registrationStorageKeyValue);
+            await TryCopyContainerAsync(_registrationgzStorageContainer, currentDate, _registrationStorageAccountName, _registrationStorageKeyValue);
+
+            Task.WaitAll(deleteTasks.ToArray());
+        }
+
+        private async Task<bool> TryCopyContainerAsync(string container, DateTime date, string sourceAccountName, string sourceAccountKey)
+        {
+            Stopwatch sw = new Stopwatch();
+            string azCopyTempFolder = $@"{Directory.GetCurrentDirectory()}\azCopy_{container}";
+            string destContainer = $"{Format(date)}-{container}";
+            string logFile = $"{destContainer}.log";
+            string azCopyLogPath = Path.Combine(azCopyTempFolder, logFile);
+            RefreshLogData(azCopyTempFolder, azCopyLogPath);
+
+            if (await TryCreateDestinationContainerAsync(destContainer, _destStorageAccountName, _destStorageKeyValue))
+            {
+                string arguments = $"/Source:https://{sourceAccountName}.blob.core.windows.net/{container}/ " +
+                                   $"/Dest:https://{_destStorageAccountName}.blob.core.windows.net/{destContainer}/ " +
+                                   $"/SourceKey:{sourceAccountKey} /DestKey:{_destStorageKeyValue} " +
+                                   $"/S /Z:{azCopyTempFolder} /V:{azCopyLogPath}";
+
+                string argumentsLog = $"/Source:{azCopyTempFolder} /Dest:https://{_destStorageAccountName}.blob.core.windows.net/logs" +
+                                      $" /DestKey:{_destStorageKeyValue} /destType:blob /Pattern:{logFile}";
+
+                try
+                {
+                    ProcessStartInfo copyToAzureProc = new ProcessStartInfo();
+                    copyToAzureProc.FileName = @"cmd.exe";
+                    copyToAzureProc.Arguments = $"/c {AzCopyPath} {arguments}";
+                    copyToAzureProc.CreateNoWindow = false;
+                    copyToAzureProc.UseShellExecute = false;
+#if Debug
+                copyToAzureProc.RedirectStandardOutput = true;
+#endif
+
+                    Logger.LogInformation($"StartContainerCopy:{container}");
+                    sw.Start();
+                    Process p = Process.Start(copyToAzureProc);
+                    p.WaitForExit();
+#if Debug
+                string result = p.StandardOutput.ReadToEnd();
+                Console.Write(result);
+#endif
+                    sw.Stop();
+                    int exitCode = p.ExitCode;
+                    Logger.LogInformation($"EndContainerCopy:{container}:{exitCode}:{sw.ElapsedMilliseconds}");
+                    p.Close();
+                }
+                catch (Exception ex)
+                {
+                    Logger.LogError($"BackupV3Storage:Exception on backup save.{ex.ToString()}.");
```

Comment:
You are encoding the type name into the log message, this should not be necessary if you use `ILogger<T>`.